lib/ex_twilio/config.ex

```diff
@@ -6,6 +6,10 @@ defmodule ExTwilio.Config do
   values from environment variables at runtime.
   """
 
+  @api_version Application.get_env(:ex_twilio, :api_version, "2010-04-01")
+  @protocol Application.get_env(:ex_twilio, :protocol, "https")
+  @workspace_sid Application.get_env(:ex_twilio, :workspace_sid, "12345")
```

Comment:
Let's not extract these to module attributes because this will require these values to be populated at compile time. This is not always the case, especially when people are using environment variables.

Reply:
When would someone pass the environment variable at runtime?

Is it for secret rotation? That should just be a new build and release.

I can switch it back, but it seems like an anti-pattern.

Reply:
It's an obscure case, but I've run into it several times. It happens when your compilation happens on a different machine than the compiler runs on. 

For example, Heroku will compile your code during the deploy, and the environment variables are not necessarily available during that compile process. Heroku then provisions the environment variables only in the runtime environment.

I agree that it isn't ideal, but I don't want to change this behavior right now as I don't have time to refactor ExTwilio to use the more idiomatic `init` callback configuration style.

Reply:
Heh... Heroku needs to pick its act up here...